doc/release-notes.md

```diff
@@ -33,6 +33,28 @@ frequently tested on them.
 Notable changes
 ===============
 
+Performance Improvements
+--------------
+
+Validation speed and network propogation performance have been greatly
+improved, leading to much shorter sync and initial block download times.
```

Comment:
Perhaps qualify this with "in some configurations"... people who are bandwidth starved or I/O bottlenecked won't need much difference.